### ISSUE TITLE
Admob Extension 1.4.0 Compatibility Issues with LTS Version

### ISSUE THREAD

**Issue by DarocMike** [YoYoGames/GMEXT-AdMob#66]
Environment:

LTS Early Look version: Runtime 2022.0.3.83, IDE v2022.0.3.71
Android Studio: Jellyfish | 2023.3.1 Patch 2 (June 10, 2024)
NDK: 26.3.11579264
(I also tried Android Studio Koala Patch 2 and the corresponding NDK, but still no luck.)
Admob Extension version: 1.4.0

I encounter a build failure with the following error:
```
> Task :com.yadda.yaddayadda:processDebugMainManifest FAILED
package="com.yadda.yaddayadda" found in source AndroidManifest.xml: C:\Users\micda\AppData\Roaming\GameMakerStudio2-LTS\Cache\GMS2CACHE\bewitchedf_87F29E5B_728A6903\android\Default\com.yadda.yaddayadda\src\main\AndroidManifest.xml.
Setting the namespace via the package attribute in the source AndroidManifest.xml is no longer supported, and the value is ignored.
Recommendation: remove package="com.yadda.yaddayadda" from the source AndroidManifest.xml: C:\Users\micda\AppData\Roaming\GameMakerStudio2-LTS\Cache\GMS2CACHE\bewitchedf_87F29E5B_728A6903\android\Default\com.yadda.yaddayadda\src\main\AndroidManifest.xml.
C:\Users\micda\AppData\Roaming\GameMakerStudio2-LTS\Cache\GMS2CACHE\bewitchedf_87F29E5B_728A6903\android\Default\com.yadda.yaddayadda\src\main\AndroidManifest.xml:32:13-58 Error:
    Attribute property#android.adservices.AD_SERVICES_CONFIG@resource value=(@xml/ga_ad_services_config) from [com.google.android.gms:play-services-measurement-api:21.5.0] AndroidManifest.xml:32:13-58
    is also present at [com.google.android.gms:play-services-ads-lite:23.1.0] AndroidManifest.xml:92:13-59 value=(@xml/gma_ad_services_config).
    Suggestion: add 'tools:replace="android:resource"' to <property> element at AndroidManifest.xml to override.

See https://developer.android.com/r/studio-ui/build/manifest-merger for more information about the manifest merger.


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':com.yadda.yaddayadda:processDebugMainManifest'.
> Manifest merger failed : Attribute property#android.adservices.AD_SERVICES_CONFIG@resource value=(@xml/ga_ad_services_config) from [com.google.android.gms:play-services-measurement-api:21.5.0] AndroidManifest.xml:32:13-58
      is also present at [com.google.android.gms:play-services-ads-lite:23.1.0] AndroidManifest.xml:92:13-59 value=(@xml/gma_ad_services_config).
      Suggestion: add 'tools:replace="android:resource"' to <property> element at AndroidManifest.xml to override.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 2s
12 actionable tasks: 1 executed, 11 up-to-date

```
I've also tried multiple older Admob Extension versions, and only version 1.2.3 works—but only as long as I don't add mediation extensions, which also causes build errors.

**Comment by DiasFranciscoA** [comment]
Hello @DarocMike thank you for reaching out to us. We were able to correctly run the demo project of the AdMob extension on the LTS version you are currently using. Probably AdMob isn't the only extension you are using. This error is due to an SDK collision which indicates that you are probably using other GooglePlayServices extensions which probably are not yet updated SDK wise so the SDKs conflict between the 2 versions. We would need a copy of the project to be able to understand the conflicts that might be happening.

**Comment by DarocMike** [comment]
How can I best send you the project? Alternatively, I can also list the extensions I have installed:
GooglePlayBilling 1.2.0
GooglePlayServices 2.1.4
FirebaseSetup 1.2.2
Firebase Analytics 2.2.3
FirebaseCloudMessaging 2.1.3
LocalNotifications 1.0.1

All of these are official YoYo extensions.

**Comment by DarocMike** [comment]
After reading your message, I disabled various extensions to see when the project builds successfully. It turns out that the project builds correctly with the AdMob extension enabled and the FirebaseSetup extensions disabled. The issue I described arises as soon as I enable both the AdMob Extension 1.4.0 and FirebaseSetup 1.2.2. When I disable the FirebaseSetup extension, everything works fine.

I then wanted to download the latest FirebaseSetup extension to check if I could run it alongside the AdMob 1.4.0 extension, but there is no newer version. This means that the project does not build when both the latest AdMob extension and the latest FirebaseSetup extension are included in the project. What should I do next?
